### PR TITLE
Fix UI breaks in mobile view

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
- [x] Video section of justly portfolio fix space.
- [x] FAQ section is not in center .
- [x] Blog section on home page fix space from top.
- [x] Fix GlassDoor alignment.

### CODE REVIEW

File: vue-frontend/src/components/jobs/FaqSection.vue [hunk 1]
```diff
@@ -22,7 +22,7 @@
         />
       </div>
       <div class="tw-flex-[67%]">
-        <div class="faq-content tw-pl-[60px]">
+        <div class="faq-content sm:tw-pl-[60px]">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/FaqSection.vue:25]: I think we need pl for large devices, in small devices it should be centered, no padding is there

File: vue-frontend/src/components/home-new/UserReview.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    class="sm:tw-container tw-relative tw-pt-[50px] tw-pb-20 md:tw-pt-[100px] md:tw-pb-[150px] lg:tw-pt-[150px] lg:tw-pb-[250px]"
+    class="sm:tw-container tw-relative tw-pt-[50px] tw-pb-[5rem] sm:tw-pb-20 md:tw-pt-[100px] md:tw-pb-[150px] lg:tw-pt-[150px] lg:tw-pb-[250px]"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/UserReview.vue:3]: Isn't it work by just making `pb` from `sm`?

File: vue-frontend/src/components/portfolio/VideoSection.vue [hunk 3]
```diff
@@ -135,7 +135,7 @@
   </section>
 
   <!-- large screen slider -->
-  <section v-else-if="response.slider" class="tw-relative">
+  <section v-else-if="response.slider" class="tw-relative tw-mt-20 lg:tw-mt-0">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/VideoSection.vue:138]: The bottom space is too much, can't we reduce `py` of the branding section?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/home-new/UserReview.vue (hunk 2 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/home-new/UserReview.vue:3]: no, it wasn't working that's why need to use sm.

File: vue-frontend/src/components/portfolio/BrandingSection.vue [hunk 4]
```diff
@@ -2,7 +2,7 @@
   <section class="tw-bg-white">
     <div
       v-if="response.solution"
-      class="tw-container tw-py-40 lg:tw-py-80 tw-relative"
+      class="tw-container tw-py-14 sm:tw-py-20 lg:tw-py-80 tw-relative"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/BrandingSection.vue:5]: `tw-py-14 sm:tw-py-20 lg:tw-py-80`
Which will be applicable?

File: vue-frontend/src/components/portfolio/BrandingSection.vue [hunk 5]
```diff
@@ -34,7 +34,7 @@
     </div>
     <div
       v-if="response.title"
-      class="tw-container tw-py-40 lg:tw-py-80 tw-relative"
+      class="tw-container tw-py-14 sm:tw-py-20 lg:tw-py-80 tw-relative"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/portfolio/BrandingSection.vue:37]: Same as above comment

File: vue-frontend/src/components/portfolio/VideoSection.vue (hunk 3 above)
Comment (cp-sumi-k) [reply vue-frontend/src/components/portfolio/VideoSection.vue:141]: No need to add or remove the padding in this section. space is due to branding section, fix their

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/portfolio/BrandingSection.vue (hunk 4 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/portfolio/BrandingSection.vue:5]: All.

Comment (cp-shruti-s) [inline vue-frontend/src/components/portfolio/BrandingSection.vue:5]: All sections should have equal spaces between them

Comment (cp-sumi-k) [reply vue-frontend/src/components/portfolio/BrandingSection.vue:5]: From small to large, `tw-py-20` will be applicable, and from lg to rest `tw-py-80` will be applicable, what is use of `tw-py-14`?

**Review by cp-sumi-k — APPROVED** [review]

**Review by cp-shruti-s — APPROVED** [review]

**Pull request merged**